javalib/src/main/scala/java/security/Permission.scala

```diff
@@ -0,0 +1,12 @@
+package java.security
+
+abstract class Permission(name: String) extends Guard with Serializable {
+  //def checkGuard(a: Any): Unit
```

Comment:
I just need the interface to let the Exception available without linking errors, I have no feasible way to implement it ...